core/templates/services/date-time-format.service.ts

```diff
@@ -45,17 +53,32 @@ export class DateTimeFormatService {
       return moment(date).format('MM/DD/YY');
     }
   }
-  // Returns date along with time.
+  /**
+   * This function converts a millisecond date to a human-readable date
```

Comment:
Could you please replace human-readable with the exact format like `'MMM D HH:mm A'`?

Reply:
> Can you please merge the latest develop with your branch once and see if the tests pass?

I have adressed your comments and merged developed with my branch. Hope it looks better!  @bansalnitish

Reply:
Done